stack-release.html

```diff
@@ -95,12 +96,13 @@
 							compareDiffAndUpdateElement(
 								"src_latest_tag_vs_main",
 								"synapse-web-monorepo",
+								synapseReactClientLatestVersionTag,
 								"main",
-								`synapse-react-client/v${synapseReactClientLatestVersion}`,
 								"Latest synapse-react-client contains all changes in main.",
-								"Latest synapse-react-client does not contain all changes in main."
+								"Latest synapse-react-client does not contain all changes in main (may include irrelevant changes, such as portals).",
+								"#ff9966"
```

Comment:
Update the message and use a red that is less "scary". This diff may not include changes that are relevant for Synapse.org. GitHub does not provide a way to query for a subset of a diff, so this is the best we can do in a single API call.